### PR TITLE
Added wrappers for Readiness and Heartbeat.

### PR DESCRIPTION
**Pull request by moiflo033**
Added wrappers for:
- types/reg/drone/service/common/Heartbeat_0_1.h
- types/reg/drone/service/common/Readiness_0_1.h

I noticed that these two wrappers were missing while implementing a Servo controller.

### CODE REVIEW

File: src/wrappers/reg/drone/service/common/Heartbeat_0_1.hpp [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+ **************************************************************************************/
+
+#include <libcanard/canard.h>
+
+#include <types/reg/drone/service/common/Heartbeat_0_1.h>
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
```
Comment (aentinger) [inline src/wrappers/reg/drone/service/common/Heartbeat_0_1.hpp:22]: Please delete one line break.

File: src/wrappers/reg/drone/service/common/Heartbeat_0_1.hpp [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+namespace reg {
+namespace drone {
+namespace service {
+namespace common {
+/**************************************************************************************
```
Comment (aentinger) [inline src/wrappers/reg/drone/service/common/Heartbeat_0_1.hpp:28]: Please add one line break 😉

File: src/wrappers/reg/drone/service/common/Readiness_0_1.hpp [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+ **************************************************************************************/
+
+#include <libcanard/canard.h>
+
+#include <types/reg/drone/service/common/Readiness_0_1.h>
+
+/**************************************************************************************
+ * NAMESPACE
+ **************************************************************************************/
+
```
Comment (aentinger) [inline src/wrappers/reg/drone/service/common/Readiness_0_1.hpp:22]: Same here with the line breaks 😉

**Review by aentinger — CHANGES_REQUESTED** [review]
Thank you very much for your contribution @moiflo033 🚀 Since its pretty much perfect I got to be a little perfection nazi and ask you to fix the linebreaks around the opening namespace section in both header files so that there is one line break before and after. After that this change is good to merge.

**Comment by moiflo033** [comment]
No worries i missed those. Do you by any chance have a formatting spec anywhere that i could use in clion? If that is the case i could maybe start work on #98.

**Review by aentinger — APPROVED** [review]

**Comment by aentinger** [comment]
> Do you by any chance have a formatting spec anywhere that i could use in clion?

Sorry, no I don't. If you come up with one though I'll be sure to accept it as a PR.

**Pull request merged**